docs/docs/general/options.md

```diff
@@ -2,9 +2,74 @@
 title: Options
 ---
 
+## Option resolution
+
+Options are resolved from top to bottom, using a context dependant route.
+
+### Chart level options
+
+* options
+* defaults.controllers[`config.type`]
+* defaults
+
+### Dataset level options
+
+`dataset.type` defaults to `config.type`, if not specified.
+
+* dataset
+* options.datasets[`dataset.type`]
+* options.controllers[`dataset.type`].datasets
+* options
+* defaults.datasets[`dataset.type`]
+* defaults.controllers[`dataset.type`].datasets
+* defaults
+
+### Dataset animation options
+
+* dataset.animation
+* options.controllers[`dataset.type`].datasets.animation
+* options.animation
+* defaults.controllers[`dataset.type`].datasets.animation
+* defaults.animation
+
+### Dataset element level options
+
+Each scope is looked up with `elementType` prefix in the option name first, then wihtout the prefix. For example, `radius` for `point` element is looked up using `pointRadius` and if that does not hit, then `radius`.
+
+* dataset
+* options.datasets[`dataset.type`]
+* options.controllers[`dataset.type`].datasets
+* options.controllers[`dataset.type`].elements[`elementType`]
+* options.elements[`elementType`]
+* options
+* defaults.datasets[`dataset.type`]
+* defaults.controllers[`dataset.type`].datasets
+* defaults.controllers[`dataset.type`].elements[`elementType`]
+* defaults.elements[`elementType`]
+* defaults
+
+### Scale options
+
+* options.scales
+* defaults.controllers[`config.type`].scales
+* defaults.controllers[`dataset.type`].scales
+* defaults.scales
+
+### Plugin options
+
+* options.plugins[`plugin.id`]
+* options.controllers[`config.type`].plugins[`plugin.id`]
+* (options.[`...plugin.additionalOptionScopes`])
```

Comment:
It could be useful to clarify what `additionalOptionScopes` is

Reply:
Yeah, the section where that could link to, is still missing :)